siliconcompiler/tools/openroad/scripts/sc_floorplan.tcl

```diff
@@ -302,7 +302,17 @@ if {[dict exists $sc_cfg constraint component]} {
 
     set inst [[ord::get_db_block] findInst $name]
     if { $inst == "NULL" } {
-      utl::error FLW 1 "Could not find instance: $name"
+      utl::warn FLW 1 "Could not find instance: $name"
+
+      if { $cell == "" } {
+        utl::error FLW 1 "Unable to create instance for $name as the cell has not been specified"
+      } else {
+        set master [ord::get_db findMaster $cell]
+        if { $master == "NULL" } {
+          utl::error FLW 1 "Unable to create $name, $cell is not a valid type"
+        }
+        set inst [odb::dbInst_create [ord::get_db_block] $master $name]
```

Comment:
Just a general question:
I guess the instance is just an instantiation of a specific cell but
What is the master?

Reply:
the master is the cell type (its what it is called in ODB)